index.js

```diff
@@ -67,4 +53,3 @@ function of (x) {
     called = true
```

Comment:
We need to tweak this function! Your PR through up the bug. `chain(of, of(1))` loops forever.

Can we have

```js
if (abort) return cb(abort)
var wasCalled = called
called = true
cb(wasCalled, x)
```

Thanks!